Troupon.Checkout.Api/Convention/PwcApiConventions.cs

```diff
@@ -0,0 +1,171 @@
+using System.Threading;
```

Comment:
@pedro-beslogic good !

I take care of having this only once in all 3 projects